cmd/deploy.go

```diff
@@ -178,6 +178,12 @@ func runDeploy(cmdCtx *cmdctx.CmdContext) error {
 			opts.DockerfilePath = filepath.Join(filepath.Dir(cmdCtx.ConfigFile), dockerfilePath)
 		}
 
+		if dockerBuildTarget := cmdCtx.Config.GetString("build-target"); dockerBuildTarget != "" {
+			opts.Target = dockerBuildTarget
+		} else if dockerBuildTarget := cmdCtx.AppConfig.DockerBuildTarget(); dockerBuildTarget != "" {
+			opts.Target = filepath.Join(filepath.Dir(cmdCtx.ConfigFile), dockerBuildTarget)
```

Comment:
Should this be the raw value from the config rather than joined with the config path?

Reply:
Doh!

Reply:
Looks good!